tutoring.md

```diff
@@ -0,0 +1,57 @@
+1. [FAQ](#faq)
+  1. [Which tutoring service should I use?](#which-tutoring-service-should-i-use)
+
+## Overview
+Ada Developers Academy is committed to ensuring that all of our students receive the support they need to succeed with our curriculum and go on to become professional software developers.
+
+To that end we offer a number avenues for accessing tutoring services outside of class hours. This document explains the different tutoring services offered including when and where they are available and has a FAQ for guiding you through the process of selecting an appropriate tutoring service.
+
+## Services
+### Open Tutoring
+Open Tutoring is our most available tutoring service and should likely be your first choice if you feel tutoring is necessary.
+
+Open Tutoring consists of a fixed set of hours during which experienced software developers are available at the Ada campus to answer questions and assist with debugging for any students present at that time.
+
+Due to the nature of Open Tutoring there aren't always enough Tutors for each student to receive 1-on-1 dedicated time through the entire session, however our Tutors attempt to make sure they work with every student at least once.
+
+The current schedule for Open Tutoring is:
+
+| Monday | Tuesday | Wednesday | Thursday | Friday | Saturday | Sunday |
+|:------:|:-------:|:---------:|:--------:|:------:|:--------:|:------:|
+|        | 5-7 PM  |           | 5-7 PM   |        | 11 - 2PM |        |
+
+(Last updated 2/23/2016 - Check your class Google calendar for temporary schedule changes.)
+
+### Ad-hoc Tutoring
+Ad-hoc Tutoring is an ideal option when you feel that more dedicated 1-on-1 instruction is necessary, or you're unable to attend the Open Tutoring hours.
+
+Ad-hoc Tutoring consists of one-time, individual tutoring sessions arranged through our mailing list (the `ada-tutors` Google group). The mailing list has many volunteers from the tech industry in Seattle, including a number of Adies from previous cohorts.
+
+When you post a message to the mailing list please include the following information to help our tutors figure out if they would be a good fit for working with you:
+- Where you are at in the curriculum, including what project you're working on if your tutoring request relates to a specific project
+- What particular technologies you're working with (Ruby, Rails, JavaScript, Backbone, etc.)
+- What times would work best for you
+- Something about yourself and/or your preferred learning and communication styles
+
+Once you've posted a message to the mailing list an Ada staff member will follow up with you within a few days to make sure that you were able to find assistance or help to make other arrangements.
+
+### Scheduled Tutoring
+Scheduled Tutoring is generally reserved for students who have need of recurrent, dedicated instruction outside of class such as when significant instruction material has been missed due to absences.
```

Comment:
I don't think the "absence" example is too good here since that's not the case for 99% of our scheduled tutoring folks. We were actually talking about the ad-hoc tutoring being for extended absences